AmericanMedicalAssociation/ama-style-guide-2
# EWL-8758: Trending tag article styling.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- N/A

**Jira Ticket**
- [EWL-8758: Trending module - style and add trending tag to article](https://issues.ama-assn.org/browse/EWL-8758)

## Description
Adds styling and images for article trending tag.


## To Test
- Checkout the following PR: https://github.com/AmericanMedicalAssociation/ama-d8/pull/2586
- link style guides
- confirm article tag displays as shown in the following designs:

Desktop https://zpl.io/25RPy5J
Tablet https://zpl.io/adwdpeL
Mobile https://zpl.io/VqeB68G

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
- N/A


## Remaining Tasks
- N/A


## Additional Notes
- AMA-d8 PR: https://github.com/AmericanMedicalAssociation/ama-d8/pull/2586

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
